src/modules/commander/Commander.cpp

```diff
@@ -3993,38 +3993,17 @@ Commander::offboard_control_update()
 		}
 	}
 
-	if (offboard_control_mode.timestamp != 0 &&
-	    offboard_control_mode.timestamp + OFFBOARD_TIMEOUT > hrt_absolute_time()) {
-		if (status_flags.offboard_control_signal_lost) {
-			status_flags.offboard_control_signal_lost = false;
-			status_flags.offboard_control_loss_timeout = false;
-			_status_changed = true;
-		}
-
-	} else {
-		if (!status_flags.offboard_control_signal_lost) {
-			status_flags.offboard_control_signal_lost = true;
-			_status_changed = true;
-		}
-
-		/* check timer if offboard was there but now lost */
-		if (!status_flags.offboard_control_loss_timeout && offboard_control_mode.timestamp != 0) {
-			if (_param_com_of_loss_t.get() < FLT_EPSILON) {
-				/* execute loss action immediately */
-				status_flags.offboard_control_loss_timeout = true;
+	_offboard_available.set_hysteresis_time_from(true, _param_com_of_loss_t.get());
```

Comment:
This should only be set on param change.

Reply:
Ok, I'll fix it.

Reply:
Done in https://github.com/PX4/Firmware/pull/14283/commits/c70929815beee67dd05a034c735a9e8bb4fdb732